rules/badges/in_game.json

```diff
@@ -86,5 +86,13 @@
     "operation": "each_greater",
     "targets": [["kano-draw","groups.pixelhack.challengeNo", 13]],
     "title": "Pixel Hack"
+  },
+  "super_user": {
```

Comment:
Can we move this one up to be with Eleanor's Saviour so that we keep the TQ badges together?

Reply:
Sure, I can move it up..